gcloud-java-dns/README.md

```diff
@@ -0,0 +1,324 @@
+Google Cloud Java Client for DNS
+================================
+
+Java idiomatic client for [Google Cloud DNS] (https://cloud.google.com/dns/).
+
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/gcloud-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/gcloud-java)
+[![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/gcloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/gcloud-java?branch=master)
+[![Maven](https://img.shields.io/maven-central/v/com.google.gcloud/gcloud-java-dns.svg)]( https://img.shields.io/maven-central/v/com.google.gcloud/gcloud-java-dns.svg)
+[![Codacy Badge](https://api.codacy.com/project/badge/grade/9da006ad7c3a4fe1abd142e77c003917)](https://www.codacy.com/app/mziccard/gcloud-java)
+[![Dependency Status](https://www.versioneye.com/user/projects/56bd8ee72a29ed002d2b0969/badge.svg?style=flat)](https://www.versioneye.com/user/projects/56bd8ee72a29ed002d2b0969)
+
+-  [Homepage] (https://googlecloudplatform.github.io/gcloud-java/)
+-  [API Documentation] (http://googlecloudplatform.github.io/gcloud-java/apidocs/index.html?com/google/gcloud/dns/package-summary.html)
+
+> Note: This client is a work-in-progress, and may occasionally
+> make backwards-incompatible changes.
+
+Quickstart
+----------
+If you are using Maven, add this to your pom.xml file
+```xml
+<dependency>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-dns</artifactId>
+  <version>0.1.4</version>
+</dependency>
+```
+If you are using Gradle, add this to your dependencies
+```Groovy
+compile 'com.google.gcloud:gcloud-java-dns:0.1.5'
+```
+If you are using SBT, add this to your dependencies
+```Scala
+libraryDependencies += "com.google.gcloud" % "gcloud-java-dns" % "0.1.5"
+```
+
+Example Application
+-------------------
+
+[`DnsExample`](../gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/DnsExample.java)
+is a simple command line interface that provides some of Google Cloud DNS's functionality.  Read
+more about using the application on the
+[`DnsExample` docs page](http://googlecloudplatform.github.io/gcloud-java/apidocs/?com/google/gcloud/examples/dns/DnsExample.html).
+
+Authentication
+--------------
+
+See the [Authentication](https://github.com/GoogleCloudPlatform/gcloud-java#authentication) section
+in the base directory's README.
+
+About Google Cloud DNS
+--------------------------
+
+[Google Cloud DNS][cloud-dns] is a scalable, reliable and managed authoritative Domain Name System
+(DNS) service running on the same infrastructure as Google. It has low latency, high availability
+and is a cost-effective way to make your applications and services available to your users. Cloud
+DNS translates requests for domain names like www.google.com into IP addresses like 74.125.29.101.
+Cloud DNS is programmable. You can easily publish and manage millions of DNS zones and records using
```

Comment:
Remove these last two lines (users know if it they're reading this README b/c they've found our library)